kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -331,4 +331,8 @@
     font-size: 14px;
   }
 
+  // .subpage-nav {
```

Comment:
What did this CSS class definition do to deserve this purgatory? Maybe just delete it if it is unneeded?